src/main.ts

```diff
@@ -116,6 +120,20 @@ export function main(command?: string) {
         ready: () => AutumnAton.available(),
         sobriety: "either",
       },
+      {
+        name: "June Cleaver",
+        completed: () => !!get("_juneCleaverFightsLeft"),
```

Comment:
Double negative?

Reply:
Oh wait, I see, that converts it to a boolean.

Reply:
This just means "non zero" in a more clever way i guess